CliFx/Services/CommandInitializer.cs

```diff
@@ -87,5 +93,52 @@ public void InitializeCommand(ICommand command, CommandSchema commandSchema, Com
                 throw new CliFxException($"One or more required options were not set: {unsetRequiredOptionNames}.");
             }
         }
+
+        private void InitializeCommandArguments(ICommand command, TargetCommandSchema targetCommandSchema)
+        {
+            if (targetCommandSchema.Schema is null)
+            {
+                throw new ArgumentException("Cannot initialize command without a schema.");
+            }
```

Comment:
Can this actually happen? `TargetCommandSchema` seems to be marked as non-nullable.

Reply:
The check is for `TargetCommandSchema.Schema`, which is nullable. See my comment above.